### PR TITLE
add license to docs

### PR DESCRIPTION
**Pull request by NoahGorny**
## Description

- docs: Add license notice to readme
- docs: Add newline in readme and index.rst

## Motivation and Context

I felt like the readme could use a notice like this, also a newline was missing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] If I have added a new file, I also added it to ``clean_files.txt`` and formatted it using ``lint_clean_files.sh``.
- [ ] I have added tests to cover my changes, and all the new and existing tests pass.

### CODE REVIEW

**Review by cornfeedhobo — APPROVED** [review]
Lgtm. Thanks!

File: docs/index.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Welcome to Bash-it's documentation!
 Includes autocompletion, themes, aliases, custom functions, a few stolen pieces from Steve Losh, and more.
 
 Bash-it provides a solid framework for using, developing and maintaining shell scripts and custom commands for your daily work.
-If you're using the *Bourne Again Shell* (Bash) regularly and have been looking for an easy way on how to keep all of these nice little scripts and aliases under control, then Bash-it is for you!
+If you're using the *Bourne Again Shell* (Bash) regularly and have been looking for an easy way on how to keep all of these nice little scripts and aliases under control, then Bash-it is for you!
```
Comment (davidpfarrell) [inline docs/index.rst:11]: Trailing whitespace?

File: docs/README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 Includes autocompletion, themes, aliases, custom functions, a few stolen pieces from Steve Losh, and more.
 
 Bash-it provides a solid framework for using, developing and maintaining shell scripts and custom commands for your daily work.
-If you're using the _Bourne Again Shell_ (Bash) regularly and have been looking for an easy way on how to keep all of these nice little scripts and aliases under control, then Bash-it is for you!
+If you're using the _Bourne Again Shell_ (Bash) regularly and have been looking for an easy way on how to keep all of these nice little scripts and aliases under control, then Bash-it is for you!  
 Stop polluting your `~/bin` directory and your `.bashrc` file, fork/clone Bash-it and start hacking away.
```
Comment (davidpfarrell) [inline docs/README.md:13]: Trailing whitespace?

**Review by davidpfarrell — COMMENTED** [review]
About time we added a license link ;)
Looks like some extra trailing whitespace made it in tho?

**Pull request merged**